content/java/concepts/strings/terms/equalsIgnoreCase/equalsIgnoreCase.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.equalsIgnoreCase()'
+Description: 'Compares strings in a case-insensitive manner.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'String'
+CatalogContent:
+  - 'learn-java'
+---
+
+The **`.equalsIgnoreCase()`** method compares strings and then then gives an output of true or false. It is not a case sensitive method.
+
+## Syntax
+
+```pseudo
+String nameOfYourString = "the value you assign";
+String nameOfYourString2 = "THE VALUE YOU ASSIGN";
+String nameOfYourString3 = "Value you assing";
+System.out.pritln(nameOfYourString.equalsIgnoreCase(nameOfYourString2)); //output of this will be true.
+System.out.pritln(nameOfYourString.equalsIgnoreCase(nameOfYourString3)); //output of this will be false.
+```
+
+## Example
+
+On the example below, we have three different strings named firstString, secondString, and thirdString. To store the result we created two boolean type variables, which are called firstResult and secondResult. With the **.equalsIgnoreCase()** method, we will be comparing the values we've assigned to strings. As a result, we will get an output of true, if the strings we compare have the same values. However, we will be getting an output of false, if the values we assigned to those strings are different. Then, as mentioned in the beginning, we will store those values in two different boolean type variables and print the result.
+
+```example
+String firstString="welcome";  //decleration of the first string.

```

Comment:
I'd remove the comments from the example. They're over-long and unnecessary with the example description.